docs/reST/ref/display.rst

```diff
@@ -42,6 +42,42 @@ user adjusts the window dimensions. Hardware displays that draw direct to the
 screen will get ``pygame.VIDEOEXPOSE`` events when portions of the window must
 be redrawn.
 
+In pygame 2 (when built with SDL 2), there is a new type of event called
```

Comment:
For the whole of the pygame documentation (and not just here) we should decide on how we talk about new things and changes. It's easiest IMHO to just talk about pygame versions, and not add anything about SDL 2. 
- Adding both is redundant most of the time.
- Probably most readers won't know what SDL 2 is, and it isn't relevant to them.
- We will not continue to support SDL1 builds of pygame, it's only useful for development.
- Keeping it to 'what pygame version do I have?' makes it simpler for people.

Reply:
I guess that throughout the docs we can replace sdl2 with pygame 2, just to make it easier for users. I will open a PR for this

Reply:
There's a separate PR ongoing for this here: https://github.com/pygame/pygame/pull/2211